examples/wait-for-domain-example.c

```diff
@@ -0,0 +1,74 @@
+/* The LibVMI Library is an introspection library that simplifies access to
+ * memory in a target virtual machine or in a file containing a dump of
+ * a system's physical memory.  LibVMI is based on the XenAccess Library.
+ *
+ * Author: Steven Maresca (steven.maresca@zentific.com)
```

Comment:
I don't think you are Steve ;)

Reply:
Ah yes, I will change that. I was copied from another test